test/test_holding_verification.py

```diff
@@ -467,7 +490,7 @@ def test_main_should_call_2_methods_once_and_run_method_twice_if_2_files_have_be
         self.assertEqual(1, db_connection.cursor.call_count)
         self.assertEqual(1, db_connection.close.call_count)
 
-    def test_main_should_call_2_methods_once_and_run_method_twice_if_a_folder_with_2_files_have_been_passed_in(self):
+    def test_main_should_call_2_methods_once_and_run_method_3x_if_a_folder_with_3_files_have_been_passed_in(self):
```

Comment:
Couple of things here. 
1. The test doesn't pass for me, the line you've added fails. The test above fails as well in a similar place. These are passing on GitHub actions but it'd be nice to get them working on all platforms to make it easier to develop. 
2. It's not really clear what these tests are doing. You're counting the method calls but it doesn't say why. Can you change it a bit so you're checking method arguments or something rather than just the count. It's not a massive problem but it is confusing.

Reply:
I'm counting method calls in `run` just in case a method is removed/duplicated (on purpose/by accident) then the test would fail.

With `mock_holding_verification.get_csv_output_writer_and_file_name_args.call_count`, I'm testing the arguments directly below, on line 505 & 506.

For `mock_holding_verification.get_file_or_dir_from_user_args`, the `run` method doesn't call it with any arguments; it has default arguments.

Maybe I should move the `run_args` check above the checks for the methods it calls? And maybe I should group stuff using some new lines?

Reply:
Yeah ok, I think rearranging it and grouping some of the bits should help :+1: